src/main/java/dev/espi/protectionstones/ListenerClass.java

```diff
@@ -55,6 +57,23 @@
 
 public class ListenerClass implements Listener {
 
+
+    @EventHandler(priority = EventPriority.LOWEST)
+    public void onBlockPlaceEvent(PlaceBlockEvent event)
+    {
+        if(ProtectionStones.getInstance().getConfigOptions().byPassWGPassthrough) {
+            var cause = event.getCause().getRootCause();
+            if (cause instanceof Player) {
```

Comment:
Would this allow players to place protection blocks in any worldguard region, even if they do not own it?

Reply:
No, worldguard checks if they are overlaping regions. If there are none, it overrides only the passthrough condition for that protection stone